### PR TITLE
[UWP Renderer]: Action/MediaEventInvoker expose a constructor that takes renderedAC

### LINKED ISSUE #6681: [UWP] `AdaptiveActionInvoker` IDL missing non-default ctor
`AdaptiveActionInvoker` has an incomplete IDL definition:

https://github.com/microsoft/AdaptiveCards/blob/f8239099c02c343a6f928ea791755b9101bd1643/source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl#L446-L451

It's missing the constructor that takes a `RenderedAdaptiveCard`:

https://github.com/microsoft/AdaptiveCards/blob/f8239099c02c343a6f928ea791755b9101bd1643/source/uwp/Renderer/lib/AdaptiveActionInvoker.h#L19

Ultimately, I think this means that `AdaptiveActionInvoker` isn't usable out of the box, but that shoul dbe investigated as well...

### PR DESCRIPTION
**Pull request by beervoley**
# Related Issue

This PR fixes #6681.

# Description
The following constructors though implemented, were not exposed through `.idl` for UWP Renderer.
` AdaptiveActionInvoker(RenderedAdaptiveCard renderResult)`
` AdaptiveMediaEventInvoker(RenderedAdaptiveCard renderResult)`

I've added these constructors to the `.idl` so they can be used when Renderer is consumed by a customer.

# How Verified
Manually verified by instantiating object in C#.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/6682)

### CODE REVIEW

**Review by paulcam206 — APPROVED** [review]
:shipit:

**Comment by almedina-ms** [comment]
is there some way to add some test in source\uwp\UWPUnitTests , adding a test that makes use of this API so that we know the API is not accidentally removed would be good enough as a first step

**Review by almedina-ms — CHANGES_REQUESTED** [review]
Can you add a unit test under source\uwp\UWPUnitTests to keep track of the API not being accidentally removed?

File: source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl [hunk 1]
```diff
@@ -446,13 +446,15 @@ namespace AdaptiveCards
             runtimeclass AdaptiveActionInvoker
             {
                 AdaptiveActionInvoker();
```
Comment (licanhua) [inline source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl:448]: Do you need to remove this?

Comment (beervoley) [reply source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl:448]: I'm not certain why? :)
Default constructor is still needed and should work as before:)

**Comment by beervoley** [comment]
> Can you add a unit test under source\uwp\UWPUnitTests to keep track of the API not being accidentally removed?

will do

**Review by licanhua — APPROVED** [review]

**Comment by beervoley** [comment]
I want to hold on to this PR before c++/winrt renderer PR is up :)
@paulcam206, do you think it's okay and can wait a little? or if not, I can add the tests and merge it today:)

**Pull request closed without merging**